### ISSUE TITLE
Conjunctival erosion

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10973]
Preferred term label:
Conjunctival erosion

Synonyms
Erosion of the conjunctiva, Conjunctival epithelial defect

Definition (free text, please give PubMed ID)
A condition characterized by the localized loss or breakdown of the conjunctival epithelium, often resulting in exposure of the underlying tissue. Conjunctival erosion may cause symptoms such as redness, irritation, foreign body sensation, and tearing. It can result from trauma, chemical injuries, ocular surface diseases, or prolonged use of contact lenses.

Parent term (use hpo.jax.org/app)
Abnormality of the conjunctiva (HP:502)

Diseases characterized by this term
Stevens-Johnson syndrome, Ocular cicatricial pemphigoid (OMIM: 164185)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001327
- label: Conjunctival erosion
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: 
- def: A condition characterized by the localized loss or breakdown of the conjunctival epithelium, often resulting in exposure of the underlying tissue. Conjunctival erosion may cause symptoms such as redness, irritation, foreign body sensation, and tearing. It can result from trauma, chemical injuries, ocular surface diseases, or prolonged use of contact lenses.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10973
- synonyms:
